client/src/components/Drawer/Drawer.js

```diff
@@ -41,8 +42,21 @@ const Drawer = ({
   isDisabled,
   successText,
   body,
-}) => (
-  <KeyboardAvoidingView
+}) => {
```

Comment:
no need to say variable - instead, stating that `\\ used to disable the success button` is sufficient